pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/rule/errorprone/xml/ProperLogger.xml

```diff
@@ -29,6 +29,7 @@ public class Foo {
         <description><![CDATA[
 Ok, special case
         ]]></description>
+        <rule-property name="loggerName">log</rule-property>
```

Comment:
This change is critical - the need to change/set the property means, the rule behaves differently, which means, we have a backwards compatibility problem....